recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+pytorch_version:
+  - 1.8.0
+  - 1.9.0
+  - 1.9.1
+  - 1.10.0
```

Comment:
I don't see any `pytorch` in conda forge pinning at https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml so I guess this is ok.

Reply:
It has been stuck in review for a while.

For builds that can support multiple pytorch versions, why not..... especially while things are in flux.

However, I do think that going a more standard approach will help conda-forge in the future.

https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1890

Reply:
Ok, let's not get blocked by https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1890 if it takes a while before being merged.

I'll open a ticket with xref to https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1890 here if we end up merging here before https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1890